app/src/components/disclosure/Disclosure.tsx

```diff
@@ -16,7 +16,10 @@ import { FlexStyleProps, SizingProps } from "../types";
 
 import { disclosureCSS, disclosureGroupCSS } from "./styles";
 
-export type DisclosureGroupProps = AriaDisclosureGroupProps;
+export type DisclosureGroupProps = AriaDisclosureGroupProps & {
+  /** If true, the DisclosureGroup will render borders between items, even the last item */
+  asSibling?: boolean;
```

Comment:
I'm not sure I fully grok this prop

Reply:
What do you think of just doing style props here. That way you can set the border?

Reply:
Done